test/jdk/sun/security/provider/acvp/Launcher.java

```diff
@@ -39,6 +39,8 @@
  * @bug 8342442 8345057
  * @library /test/lib
  * @modules java.base/sun.security.provider
+ * @run main Launcher
+ * @run main/othervm -Xcomp Launcher
```

Comment:
Thank you for adding this case. Please add it as a separate testcase:
/*
 * @test 
 * @summary Test verifies intrinsic implementation. 
 * @library /test/lib
 * @modules java.base/sun.security.provider
 * @run main/othervm -Xcomp Launcher
 */

Reply:
Done.